### ISSUE TITLE
Tratamento de viadutos (e pontes e túneis?) na altimetria

### ISSUE THREAD

**Issue by Uragawa** [biciguia/biciguia-roteirizador#2]
Resumo: Os dados da SRTM não são suficientes para representar situações de "múltiplas" elevações como um viaduto, é preciso contornar essa limitação.

Exemplo e detalhes: Ao traçar uma rota saindo da Praça da Sé até algum ponto da rua Boa Vista depois do viaduto sobre a General Carneiro, o perfil de elevação é plotado como se o viaduto não existisse (ver screenshot). Os dados da SRTM não estão _errados_: a elevação real é a da General Carneiro. Mas como existe um viaduto, quem vem pela Boa Vista não acompanha a elevação real. Viadutos pequenos provavelmente podem ser aproximados como curvas no plano entre as duas pontas, mas casos com rampas de acesso ou como o Minhocão requerem mais do que verificar se a via é fora de nível.

![screen shot 2015-06-22 at 7 15 58 pm](https://cloud.githubusercontent.com/assets/4106283/8294452/35e9ded0-1913-11e5-9680-dd65b6f8604d.png)

**Comment by tiagofassoni** [comment]
Makes sense. Provavelmente vamos ter que fazer um bug report disso no próprio graphhopper, seria uma ótima issue para subir pro upstream
